### DISCUSSION TITLE
Is there a way to tell if the site is blocked?

### DISCUSSION THREAD

**Discussion by bazmattaz** [hemantapkh/1337x#3] [Q&A] [answered]
Hi there, love this project! Is there a way to tell if the site is blocked? when making this call;

`torrents = py1337x(proxy='1337x.tw', cache='/home/user/Desktop/cache', cacheTime=500)`

It would be nice if there was a way to get the status of a connection to see if it failed. Is that possible?

**Comment by hemantapkh** [comment, marked as answer]
Currently it's not possible . You can open the site in the browser or send curl request to the site and see the content to know if the site is blocked or not.
